rebound/librebound.py

```diff
@@ -314,15 +315,48 @@ def save(self, filename):
     def load(self, filename):
         self.clibrebound.input_binary(c_char_p(filename.encode("ascii")))
         
+# Integrator Flags
+    @property 
+    def integrator_whfast_corrector(self):
+        return c_int.in_dll(self.clibrebound, "integrator_whfast_corrector").value
+    
+    @integrator_whfast_corrector.setter 
+    def integrator_whfast_corrector(self, value):
+        c_int.in_dll(self.clibrebound, "integrator_whfast_corrector").value = value
+
+    @property
+    def recalculate_jacobi_every_timestep(self):
+        return c_int.in_dll(self.clibrebound, "integrator_whfast_recalculate_jacobi_every_timestep").value
+
+    @recalculate_jacobi_every_timestep.setter
+    def recalculate_jacobi_every_timestep(self, value):
+        c_int.in_dll(self.clibrebound, "integrator_whfast_recalculate_jacobi_every_timestep").value = value
 
+    @property
+    def synchronize_every_timestep(self):
+        return c_int.in_dll(self.clibrebound, "integrator_whfast_synchronize_every_timestep").value
+
+    @synchronize_every_timestep.setter
+    def synchronize_every_timestep(self, value):
+        c_int.in_dll(self.clibrebound, "integrator_whfast_synchronize_every_timestep").value = value
+
+    @property
+    def particles_modified(self):
```

Comment:
Would it make sense to call `particles_modified` something like `integrator_whfast_recalculate_jacobi_this_timestep`?